### ISSUE TITLE
autoscaling instances will not get deleted on "terraform destroy"

### ISSUE THREAD

**Issue by JustinGuese** [kube-hetzner/terraform-hcloud-kube-hetzner#445]
autoscaling instances will not get deleted on "terraform destroy"

happened two times, so probably a small bug. nothing urgent I guess since I deleted it manually

**Comment by mysticaltech** [comment]
@ifeulner Just FYI. 

I agree with @captnCC that for now, manual deletion seems the only option. But good to keep that issue in the back of our minds.

**Comment by ifeulner** [comment]
Yes, absolutely. The only way I do see here is to do some work around (e.g. via hcloud cli, labels), but terraform does not know about the dynamically created resources.

**Comment by mysticaltech** [comment]
Exactly!

**Comment by captnCC** [comment]
The only possible way would be to deploy a the ccm/auto scaler via terraform resources, so they could be destroyed by terraform. But that adds it's own set of problems

**Comment by mysticaltech** [comment]
Exactly, so not an option.

Folks, I actually still never used the autoscaler, so I need some guidance here. In order to clarify the "Takedown" section in the readme, what is the optimal way to remove all autoscaled nodes? 

Would changing the autoscaler nodepool definitions like setting min nodes and max nodes to 0 and applying again work? Would that successfully instruct the Kube autoscaler to drain the nodes and then remove them?
